recipes/hmat-oss/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: hmat-oss
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/jeromerobert/hmat-oss/archive/{{ version }}.tar.gz
+  sha1: fe52fa22e413be862bec1b44a2b695a566525138
+
+build:
+  number: 200
+  skip: true  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - cmake
+    - blas 1.1 {{ variant }}
+    - openblas
+
+  run:
+    - blas 1.1 {{ variant }}
+    - openblas
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/hmat/hmat.h  # [unix]
+    - test -f ${PREFIX}/lib/libhmat-oss.so  # [linux]
+    - test -f ${PREFIX}/lib/libhmat-oss.dylib  # [osx]
+
+about:
+  home: https://github.com/jeromerobert/hmat-oss
+  license: GNU General Public License (GPL)
```

Comment:
Please specify GPL version. In this case, it looks like this could be replaced by `GPL2` if I'm not mistaken.